LAYOUT_ENGINE.md

```diff
@@ -0,0 +1,187 @@
+#  The Kedro Viz Graph Layout Engine
+
+An engine for drawing [Kedro](https://github.com/quantumblacklabs/kedro) pipelines, visualised as a graph built from nodes connected by edges. 
+
+The solution developed for the [Kedro Viz](https://github.com/quantumblacklabs/kedro-viz) project can quickly find high quality graph drawings while performing well on Kedro scale projects.
+
+You can see and interact with rendered graphs on the [Kedro Viz demo](https://quantumblacklabs.github.io/kedro-viz/) and explore the [code](https://github.com/quantumblacklabs/kedro-viz/tree/main/src/utils/graph) of the approach discussed here.
```

Comment:
Again, it's a bit passive "approach discussed here". And I'm not sure what "approach discussed here" is referring to.

```suggestion
The engine allows you to interact with rendered graphs on the [Kedro Viz demo](https://quantumblacklabs.github.io/kedro-viz/) . You can explore the [code](https://github.com/quantumblacklabs/kedro-viz/tree/main/src/utils/graph) of the approach discussed here.
```